python/torch_mlir/dialects/torch/importer/jit_ir/build_tools/torch_ods_gen.py

```diff
@@ -232,7 +232,8 @@ def emit_with_mutating_variants(key, **kwargs):
         is_functional_op = overload == "functional"
         emit_op(registry.get_by_triple((ns, unqual + "_", overload if not is_functional_op else "")),
                 emitter_td,
-                traits=["IsTrailingUnderscoreInplaceVariant"] if not is_functional_op else [])
+                traits=["IsTrailingUnderscoreInplaceVariant"] if not is_functional_op else [],
```

Comment:
@ramiro050 With the way `emit_with_mutating_variants` works, adding canonicalization patterns for an underscore variant op also adds it to the original op. Should we change this or is it okay to add all these empty functions?

Reply:
Hm, I think for now you can just use two `emit`s: one for `copy` and one for `copy_`, so that only `copy_` gets the canonicalizer. I would also add a `TODO` above the two `emit`s saying that we should handle specifying canonicalizer for a single variant when using `emit_with_mutating_variants`